src/modules/storage.js

```diff
@@ -0,0 +1,87 @@
+import Tasks from './tasks.js';
+
+const tasksObject = new Tasks();
+const tasks = document.getElementById('to-do-list');
+export default tasksObject;
+
+const dataStore = () => {
+  localStorage.setItem('data', JSON.stringify(tasksObject.taskList));
+};
+
+export const getCheckers = () => {
```

Comment:
- [ ] When I complete a task by clicking on a checkbox and reloading the page, the task becomes uncompleted(uncheck.). Let's kindly ensure that all actions are updated in the local storage.